qa/rpc-tests/test_framework/util.py

```diff
@@ -154,9 +154,9 @@ def initialize_chain(test_dir):
 
         # Create a 200-block-long chain; each of the 4 nodes
         # gets 25 mature blocks and 25 immature.
-        # blocks are created with timestamps 10 minutes apart, starting
-        # at 1 Jan 2014
-        block_time = 1388534400
+        # blocks are created with timestamps 10 minutes apart
+        # going backwards in time from now
```

Comment:
Ok that should do it,

pushed and travis is good

have a look when you can.

 On 03/12/2015 3:34 PM, MarcoFalke wrote:
>
> In qa/rpc-tests/test_framework/util.py
> <https://github.com/bitcoin/bitcoin/pull/7164#discussion_r46629519>:
>
> > @@ -154,9 +154,9 @@ def initialize_chain(test_dir):
> >  
> >          # Create a 200-block-long chain; each of the 4 nodes
> >          # gets 25 mature blocks and 25 immature.
> > -        # blocks are created with timestamps 10 minutes apart, starting
> > -        # at 1 Jan 2014
> > -        block_time = 1388534400
> > +        # blocks are created with timestamps 10 minutes apart
> > +        # going backwards in time from now
>
> Nit: ", starting 200 minutes in the past"?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bitcoin/bitcoin/pull/7164/files#r46629519>.
>